backend/kesaseteli/applications/api/v1/views.py

```diff
@@ -21,14 +23,33 @@
 from applications.api.v1.serializers import (
     ApplicationSerializer,
     AttachmentSerializer,
+    SchoolSerializer,
     SummerVoucherSerializer,
     YouthApplicationSerializer,
 )
 from applications.enums import ApplicationStatus
-from applications.models import Application, SummerVoucher, YouthApplication
+from applications.models import Application, School, SummerVoucher, YouthApplication
 from common.utils import DenyAll
 
 
+class SchoolListView(ListAPIView):
+    # PostgreSQL specific functionality:
+    # - Custom sorter for name field to ensure finnish language sorting order.
+    # - NOTE: This can be removed if the database is made to use collation fi_FI.UTF8
+    _name_fi = Func(
+        "name",
+        function="fi-FI-x-icu",  # fi_FI.UTF8 would be best but wasn't available
+        template='(%(expressions)s) COLLATE "%(function)s"',
+    )
+
+    queryset = School.objects.all().order_by(_name_fi.asc())
+    serializer_class = SchoolSerializer
+
+    def get_permissions(self):
```

Comment:
Can you use class attribute permission_classes here? Should be much shorter
```
class SchoolListView(ListAPIView):
    # PostgreSQL specific functionality:
    # - Custom sorter for name field to ensure finnish language sorting order.
    # - NOTE: This can be removed if the database is made to use collation fi_FI.UTF8
    _name_fi = Func(
        "name",
        function="fi-FI-x-icu",  # fi_FI.UTF8 would be best but wasn't available
        template='(%(expressions)s) COLLATE "%(function)s"',
    )

    queryset = School.objects.all().order_by(_name_fi.asc())
    serializer_class = SchoolSerializer
    permission_classes = [AllowAny]
```

Reply:
I tried that initially and doublechecked it just now. Using the permission_classes = [AllowAny] gives 403 Forbidden when accessing https://localhost:8000/v1/schools/ so no or not only.